### DISCUSSION TITLE
Filter by status in dataset items table

### DISCUSSION THREAD

**Discussion by FrankyCTY** [langfuse/langfuse#3818] [Ideas]
### Describe the feature or potential improvement

Currently LangFuse support archiving dataset items, but the archived dataset items becomes nosies on the table especially there seems to be lack of filtering or sorting in the dataset items table.

I think at least allowing users to filter by status so that they can focus on the active items make sense.

I have checked the public API, and it is the same case for `get /api/public/dataset-items` endpoint.   https://api.reference.langfuse.com/#get-/api/public/dataset-items:~:text=DATASETITEMS-,Get%20dataset%20items,-GET%20/api/public

![image](https://github.com/user-attachments/assets/e05eac9e-93c3-4323-878a-debee07cefc6) (Edited screenshot)

**Comment by FrankyCTY** [comment]
I am happy to contribute to support this feature to allow filter by status if needed.

**Comment by marcklingen** [comment]
porbably just allowing for deletions makes sense here, thanks for raising this!
